android_webview/test/shell/tango/jni/MathUtils.cpp

```diff
@@ -103,7 +104,87 @@ glm::mat4 mat4FromTranslationOrientation(const double *translation,
   return m;
 };
 
-float rayIntersectsPlane(const glm::vec3 &planeNormal, const glm::vec3 &planePosition,
+void translationOrientationAndScaleFromMat4(const glm::mat4& matrix, 
+    glm::vec3& translation, glm::quat& orientation, glm::vec3& scale) {
+
+  const float* pMatrix = glm::value_ptr(matrix);
```

Comment:
this should be p_matrix or matrix_values or something.

All the variable names in this function (and all our c++ moving forward) should use underscore_case, not camelCase

Reply:
Will change it. Thanks.